File: CA3/Attempt2/csharpVersion/CA3calculator/CA3calculator/calculator_app.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;



namespace CA3calculator
{
    class calculator_app
    {

        calculator my_calc = new Calculator();

        //#61. print the menu for the user when the program starts
        return my_calc.menu();

        string a;

        //#62. ask the user for their first number
        while a != 'E'
    
             Console.WriteLine(Console.ReadLine("Enter the first number: "));
      
            if (a == 'E')
            {

                Console.WriteLine("Goodbye. Call Again");
            }
            else (a.isdigit())
            {
                store_inputA = float(a)
                print store_inputA
                o = raw_input("Enter the operation you want: ")
                store_operation = o
                Console.WriteLine("Answer: " + str(my_calc.run_calc(store_inputA, store_operation)));
            }
            else:
               Console.WriteLine( "That wasn't a number. Try again: ");

            //#63. run - error comes back saying my_calc has no instance attribute of 'run_calc' method. therefore go to calculator file and define it.






    }
}

File: CA3/Attempt2/calculator_app.py
# -*- coding: utf-8 -*-
"""
Created on Fri Jan 26 14:42:04 2018

@author: David Doyle
"""
from calculator import Calculator

my_calc = Calculator()

#61. print the menu for the user when the program starts
my_calc.menu()
a = ''

#62. ask the user for their first number
while a != 'E':
    
    a = raw_input("Enter the first number: ")
      
    if a == 'E':
        print "Goodbye. Call Again"
    
    elif a.isdigit():
        store_inputA = float(a)
        print store_inputA
        o = raw_input("Enter the operation you want: ")
        store_operation = o
        print "Answer: " + str(my_calc.run_calc(store_inputA, store_operation))
    
    else:
        print "That wasn't a number. Try again: "
    
    #63. run - error comes back saying my_calc has no instance attribute of 'run_calc' method. therefore go to calculator file and define it.

         





